### PR TITLE
Storm's triangle implementation

### PR DESCRIPTION
**Pull request by escalonn**
talk about the PR here, justify it

### CODE REVIEW

File: TriangleLib/TriangleLib/Triangle.cs [hunk 1]
```diff
@@ -4,5 +4,150 @@ namespace TriangleLib
 {
     public class Triangle
     {
+		int Aside;
+		int Bside;
+		int Cside;
```
Comment (escalonn) [inline TriangleLib/TriangleLib/Triangle.cs:9]: doesn't follow naming convention, should be e.g. `_aSide`

**Review by escalonn — COMMENTED** [review]
address style issues

File: TriangleLib/TriangleLib/Triangle.cs (hunk 1 above)
Comment (stormrubio3000) [reply TriangleLib/TriangleLib/Triangle.cs:9]: You right.

**Comment by escalonn** [comment]
addresses issue #7

**Pull request merged**